tensorflow/java/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow;
+
+import java.util.Set;
+
+/**
+ * SavedModel representation once the SavedModel is loaded from storage.
+ */
+public class SavedModelBundle implements AutoCloseable {
```

Comment:
Ah, righto. And even though that is in a separate package, it would mean that any code using both will have to spell out the fully qualified class name (`org.tensorflow.SavedModel` vs `org.tensorflow.framework.SavedModel`). 

Okay, let's keep this as `SavedModelBundle` then. Thanks.